virtual-organizations/LIGO.yaml

```diff
@@ -359,6 +359,107 @@ DataFederations:
           - SINGAPORE_INTERNET2_OSDF_CACHE
           - SPRACE_OSDF_CACHE
         Writeback: https://origin-staging.ligo.caltech.edu:1095
+      - Path: /igwn/test
+        Authorizations:
+          - FQAN: /osg/ligo
+          - FQAN: /virgo
+          - FQAN: /virgo/virgo
+          - FQAN: /virgo/ligo
+          # No Scitokens issuers because they are duplicated with the /user/ligo path.
+          # As long as the caches are kept in sync, this below fix should be fine.
+          # But the problem should be solved either by topology (ideally) or the SciTokens XRootD plugin.
```

Comment:
Is this comment accurate? I see 3 Scitokens issuers below...

Reply:
No, I think that got grabbed from copy-and-paste. It's also above for all of our other origins I think. Maybe it was a different attempt to get around needing all base paths that an issuer would support? It looks like it first appears with the `/igwn/virgo` namespace, which also has (two) SciTokens issuers.